leaflet-treering.js

```diff
@@ -757,6 +757,12 @@ function MarkerIcon(color, imagePath) {
 function MouseLine (Lt) {
   this.layer = L.layerGroup().addTo(Lt.viewer);
   this.active = false;
+  this.entireScreenLength = false;
```

Comment:
Is this being used anywhere?

Reply:
Nope, it's a left over object I forgot to delete.